iit/model_pairs/base_model_pair.py

```diff
@@ -8,6 +9,7 @@
 from torch.utils.data import DataLoader
 from tqdm import tqdm # type: ignore
 from transformer_lens.hook_points import HookedRootModule, HookPoint # type: ignore
+from IPython.display import clear_output
```

Comment:
Why is this needed here? Don't think it is being used...

Reply:
It's not! Good catch, that was leftover from getting tqdm stuff working.